web_notify/models/res_users.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, fields, models, _
+
+
+class ResUsers(models.Model):
+
+    _inherit = 'res.users'
+
+    @api.multi
+    @api.depends('create_date')
+    def _compute_channel_names(self):
+        for record in self:
+            res_id = record.id
+            record.notify_info_channel_name = 'notify_info_%s' % res_id
+            record.notify_warning_channel_name = 'notify_warning_%s' % res_id
+
+    notify_info_channel_name = fields.Char(
+        compute='_compute_channel_names', store=True)
+    notify_warning_channel_name = fields.Char(
+        compute='_compute_channel_names', store=True)
```

Comment:
I'd think the perf gain is negligible and not worth an additional database column.